xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/usersandgroups/usersandgroups.js

```diff
@@ -55,28 +64,51 @@ window.MSCheckbox = Class.create({
     ];
     this.labels = ['','',''];
 
+    
+    
+    this.button = document.createElement("button");
+    this.button.className = "rights-edit";
+    this.button.onclick = this.createClickHandler(this);
+    
+    var img = document.createElement("img");
+    
+    this.button.appendChild(img);
+    
+    this.label = document.createElement('span');
+    this.label.hidden = true;
+    this.label.appendChild(document.createTextNode(this.labels[this.state]));
```

Comment:
The label element was present in the structure, however it was never used.
I changed the way the table was generated, so in the case one wanted to use it someday, I had to also change the way the label element was instantiated (and updated).

We should be able to remove it without any changes, it remains hidden.

Reply:
I would actually be inclined to just remove these label elements as with the alt text and title they are quite redundant though it could be considered that they are part of the API of XWiki which shouldn't be changed. Any other opinions on this?

Reply:
I don't think this label was ever used, so I'm in favor or removing it.